### PR TITLE
[stable/jenkins] Allow configuring terminationGracePeriodSeconds

### PR DESCRIPTION
**Pull request by joceluss**
#### Is this a new chart: 
No

#### What this PR does / why we need it:
Allow configuring terminationGracePeriodSeconds for the master

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/jenkins/templates/jenkins-master-deployment.yaml [hunk 1]
```diff
@@ -62,6 +62,9 @@ spec:
       {{- if .Values.master.affinity }}
       affinity:
 {{ toYaml .Values.master.affinity | indent 8 }}
+      {{- end }}
+      {{- if quote .Values.master.terminationGracePeriodSeconds }}
```
Comment (torstenwalter) [inline stable/jenkins/templates/jenkins-master-deployment.yaml:66]: Why do you use quote?

**Review by torstenwalter — COMMENTED** [review]
Can you add an entry in changeling?

File: stable/jenkins/templates/jenkins-master-deployment.yaml (hunk 1 above)
Comment (joceluss) [reply stable/jenkins/templates/jenkins-master-deployment.yaml:66]: > Can you add an entry in changeling?

Done

Comment (joceluss) [reply stable/jenkins/templates/jenkins-master-deployment.yaml:66]: > Why do you use quote?

While testing with value 0, the value was not inserted.  It turns out that without the `quote` the if statement interprets the 0 value as False and it's not inserted.

Comment (torstenwalter) [reply stable/jenkins/templates/jenkins-master-deployment.yaml:66]: Thanks for the explanation.

**Review by torstenwalter — APPROVED** [review]
/lgtm

**Comment by torstenwalter** [comment]
@joceluss Thank you for the PR!

**Comment by torstenwalter** [comment]
looks like we have an issue with plugin versions which needs to be fixed to make ci build pass again.

**Comment by torstenwalter** [comment]
@joceluss I created a PR to update the plugins in master branch, which should fix the CI build. Could you rebase you changes onto that once it got merged? Sorry that it took me so long to react.

**Comment by joceluss** [comment]
> @joceluss I created a PR to update the plugins in master branch, which should fix the CI build. Could you rebase you changes onto that once it got merged? Sorry that it took me so long to react. 

No problem, I did the rebase.

**Review by torstenwalter — APPROVED** [review]
/lgtm

**Pull request merged**